proteus/Domain.py

```diff
@@ -17,6 +65,19 @@ def __init__(self,nd,name="defaultDomain",units="m"):
         self.units=units
         self.x=[]#minx,miny,minz
         self.L=[]#bounding box when self.x is origin
+        self.g=None
+        # list of shape instances automatically attached to domain
+        self.shape_list = []
+        # list of auxiliaryVariables automatically attached to domaim
+        self.auxiliaryVariables = []
+        # attach a Mesh class
+        self.Mesh = Mesh(self)
+        # For absorption/generation zones:
+        self.porosityTypes = None
+        self.dragAlphaTypes = None
+        self.dragBetaTypes = None
+        self.epsFact_solid = None
+
```

Comment:
We don't want to mix physics-specific  quantities like g, porosity, etc. with the domain geometry.